### PR TITLE
Beginning a Project

### PR DESCRIPTION
**Pull request by acacheung**

### CODE REVIEW

File: uxd/beginning-a-project.md [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+
+## Things to Ask Client
+
+* Is this project going to be responsive? If yes, what are the min and
+  max px we should test for?
+* What browsers should we support?
+* Are there specific devices we should support?
+* Do we have to prepare for translation to other languages?
+* How do you want us to prepare the deliverables? (Zip file, GitHub, etc.)
+* Can we use BEM? Somtimes in projects that have already begun,
```
Comment (Ctannerweb) [inline uxd/beginning-a-project.md]: Include Link to BEM example you want them to see. 

http://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/
or
https://en.bem.info/method/definitions/

File: uxd/beginning-a-project.md [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+
+    i {
+      font-style: italic;
+    }
+
+    b {
+      font-weight: $bold;
+    }
+
+    .t-display {
```
Comment (Ctannerweb) [inline uxd/beginning-a-project.md]: Include and example of modified t-display class

File: uxd/beginning-a-project.md [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+* Does the client have licenses to all the fonts being used? 
+
+## Example File Structure
+
+    stylesheets
+      modules/
+        buttons.scss
+        footer.scss
+        header.scss
+      app.scss
```
Comment (Ctannerweb) [inline uxd/beginning-a-project.md]: Good idea to show how you would set up your app.scss file? Showing what your hierarchy is for the @import's.

Comment (acacheung) [inline uxd/beginning-a-project.md]: Ok sounds good

**Pull request merged**